src/main/java/com/healthapp/project/Service/UserService.java

```diff
@@ -3,32 +3,33 @@
 import com.healthapp.project.DAO.UserDAO;
 import com.healthapp.project.Entity.User;
 import java.util.Collection;
+import javax.sql.DataSource;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.jdbc.core.JdbcTemplate;
 import org.springframework.stereotype.Service;
 
-import javax.sql.DataSource;
-
 @Service
 public class UserService {
 
-    private JdbcTemplate jdbcTemplate;
-  @Autowired private UserDAO userDAO;
+   private JdbcTemplate jdbcTemplate;
+   private UserDAO userDAO;
 
-  @Autowired
-  public UserService(DataSource dataSource){
+   @Autowired
+   public UserService(DataSource dataSource, UserDAO userDAO){
      this.jdbcTemplate = new JdbcTemplate(dataSource);
-  }
+     this.userDAO = userDAO;
+   }
+  public boolean isDBConnected() {
+    try {
+      int rowCount = jdbcTemplate.queryForObject("SELECT COUNT(*) FROM user_dex", Integer.class);
+      System.out.println(rowCount);
+      return true;
+    } catch (Exception e) {
+      System.out.println(e.getMessage());
+      e.printStackTrace();
```

Comment:
The remote db currently has 1 table in it called user_dex with maybe 2 rows inside. In order to visually see my results.. i query that table for all its rows and i print the amount of rows to the console. e.printStackTrace() shows me the entire pipeline of the error itself and where it's breaking pretty much.